rest_framework_simplejwt/authentication.py

```diff
@@ -16,6 +17,19 @@
 )
 
 
+def enforce_csrf(request):
+    """
+    Enforce CSRF validation.
+    """
+    check = CSRFCheck()
+    # populates request.META['CSRF_COOKIE'], which is used in process_view()
+    check.process_request(request)
```

Comment:
# Behaviour
On this line the `request.META.['CSRF_COOKIE']` is set from `CSRF_COOKIE`. 

Later on line `csrf_token = request.META.get('CSRF_COOKIE')` it's set on the method `CSRFCheck.process_view()`.

Later `request_csrf_token` is set from  `request.META.get(settings.CSRF_HEADER_NAME, '')` (header `X-CSRFToken`).

After we have these csrf tokens we compare `request_csrf_token` against `csrf_token`.

# Problem?
My `request_csrf_token` is `None` if I don't send the header `X-CSRFToken`. So I need to send the header `X-CSRFToken` in my frontend. To do so my frontend application need to access the CSRF cookie. 

Is this supposed to happen? 

Or should my backend set `request.META.get(settings.CSRF_HEADER_NAME, '')` from `CSRF_COOKIE`?

# Note
When my frontend and backend are running from different servers my frontend doesn't have access to `CSRF_COOKIE`

Reply:
Hmm, try setting: `CSRF_COOKIE_HTTPONLY = True`